src/zope/app/server/tests/test_zpasswd.py

```diff
@@ -1,7 +1,20 @@
 import unittest
+import warnings
+from contextlib import contextmanager
 
 
 class TestZpasswdDeprecation(unittest.TestCase):
 
+    if not hasattr(unittest.TestCase, 'asserWarns'):
```

Comment:
Sorry for being late to the party but I spotted a typo here:

`asserWarns` -> `assertWarns`

Reply:
oops! :)

thank you

Reply:
Fixed in https://github.com/zopefoundation/zope.app.server/commit/e0734fdc7327a1b41542b664eb745fa4299c2a57